nextflow.config

```diff
@@ -36,7 +36,7 @@ params {
     amp_skip_ampir                  = false
     amp_ampir_model                 = 'precursor'
     amp_ampir_minlength             = 10
-    amp_ampir_minprobability        = 0.8
+    amp_ampir_minprobability        = 0.0
```

Comment:
Why have you changed this?

Reply:
Its the same idea as i described above. To get all possible predictions and have one threshold applied- at the summary part of the pipeline

Reply:
Ok! Reasoning makes sense - however I think in this case we should remove this parameter entirely so the user doesn't touch it at all, and replace it with a global `amp_minprobability` flag which applies to all tools (through AMPCombi)

Reply:
@jfy133  Ampir by default has a min probability of 0.8 as far as i remember, so to avoid that we should set it at 0.0. No?

Reply:
Yes yes! Just not here in the parameters of the `nextflow.config`,as then someone could change this ith `--amp_ampirminprobability`, which we _don't_ want them to do. So you should embeed it directly in the `ext.args` in `modules.config` fo rhte module

Reply:
Alright @jfy133, this parameter is part of the AMPir module so i added the '0.0' to the module input parametrs in amp.nf instead.